src/polling.coffee

```diff
@@ -28,7 +28,7 @@ exports.registerPollingChannel = (channel, callback) ->
       request options, ->
         done()
 
-    exports.agendaGlobal.every channel.pollingSchedule, "polling-job-#{channel._id}"
+    exports.agendaGlobal.every channel.pollingSchedule, "polling-job-#{channel._id}", null, { timezone: utils.serverTimezone() }
```

Comment:
Is there no way to rather set the agenda timezone globally? It seems like it could be error prone to remember to add this whenever we add a new task.